.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "5.0"
+  - "6.0"
```

Comment:
Could you document the supported node versions in the `README.md` and `CHANGELOG.md`?